cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Conway/DRepRetirement.hs

```diff
@@ -64,7 +57,7 @@ hprop_drep_retirement = H.integrationRetryWorkspace 2 "drep-retirement" $ \tempA
   TestnetRuntime
     { testnetMagic
     , poolNodes
-    , wallets
+    , wallets=wallet0:_
```

Comment:
This can still fail right?  What happens when it does?  Do you get nice output?

Reply:
It just prints:
```
            Pattern match failure in 'do' block at test/cardano-testnet-test/Cardano/Testnet/Test/SubmitApi/Babbage/Transaction.hs:(68,3)-(73,5)
```
This is a testnet setup so it's pretty easy to spot on when working on a test.

Reply:
Ideally we should use a dependent list here.